src/client/pages/ConnectPayment/components/useAdyenCheckout.tsx

```diff
@@ -86,17 +76,6 @@ export const useAdyenCheckout = ({
   const trackOfferEvent = useTrackOfferEvent()
 
   const successMessage = textKeys.CHECKOUT_PAYMENT_ADYEN_SETUP_DONE_MESSAGE()
-  // @ts-ignore only need to clean up timeout
-  useEffect(() => {
-    if (isSuccess && adyenState === 'MOUNTED') {
-      // Delay success message until dropin is fully loaded
-      const timeout = setTimeout(() => {
-        dropinComponent?.setStatus('success', { message: successMessage })
-      }, 300)
-
-      return () => clearTimeout(timeout)
-    }
-  }, [isSuccess, dropinComponent, adyenState, successMessage])
 
```

Comment:
We don't need this anymore? Maybe worth mentioning in the PR description in that case :)

Reply:
this was used in order to display the success message when the user refreshes the page. as we now keep that info on the localStorage my thought is we can remove it.